### ISSUE TITLE
MultiComboBox: a11y issue - Required ARIA attribute not present: aria-controls on opened MultiComboBox

### ISSUE THREAD

**Issue by fushizen** [SAP/ui5-webcomponents#10058]
### Describe the bug

Using axe devtools (a plugin used by the internal A11y team to check for issues) there is an issue with critical impact for the MultiComboBox when it is in the opened state. The issue detected by the tool is:
`Required ARIA attribute not present: aria-controls`

![image](https://github.com/user-attachments/assets/83fa1919-979c-4428-8c00-3a0b16342163)

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v1/?path=/story/inputs-multicombobox--default

### Reproduction steps

1. Go to [MCB storybook](https://sap.github.io/ui5-webcomponents-react/v1/?path=/story/inputs-multicombobox--default) and expand the drop-down selection
2. Open axe devtools in dev console
3. Issue is highlighted for the opened MultiComboBox
![image](https://github.com/user-attachments/assets/8218611d-a49c-4d01-a31f-f771d7a764c8)

### Expected Behaviour

Component to not have a11y issues detectable by the tool

### UI5 Web Components for React Version

v1.29.x

### UI5 Web Components Version

v1.29.x

### Browser

Edge

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, 
For the MCB the aria-controls property is missing when the suggestions are expanded. 

Kind regards, 

GV

**Comment by ivoplashkov** [comment]
Hello @fushizen 

The issue is already reported: https://github.com/SAP/ui5-webcomponents/issues/3827

We are aware of this issue and other issues, related to setting relationships between accessible elements via IDrefs. Setting IDRef relationships are not possible in versions <1.24 due to the nature of the custom elements and their shadow DOM. The issue is present not only for UI5 Web Components, but for every framework creating custom elements and using shadow DOM. This is a gap in the accessibility support which was solved with the introduction of the popover API - https://developer.mozilla.org/en-US/docs/Web/API/Popover_API, therefore the issue is not longer reproducible in versions 2.0 and above where the API has been adopted.

What is more, aria-controls does not affect the speech output and there will be no change in the usage of the element with or without the attribute and screen reader users will not be affected.

Thanks for understanding!

Kind Regards,
Ivaylo
